src/deploy.rs

```diff
@@ -60,7 +67,9 @@ impl DeployCmd {
                 BinaryOption::Versioned { .. } => false,
             }
         };
-        self.create_infra(build_custom_binaries)
+
+        let is_fresh_network = self.bootstrap_peer.is_none();
```

Comment:
Got two variables doing the same thing here `is_fresh_network` and `this_is_a_new_network`